src/pages/[...slug].tsx

```diff
@@ -69,7 +70,7 @@ export async function getStaticProps(context) {
 
   return {
     props: {
-      resource,
+      resource: JSON.parse(JSON.stringify(resource)),
```

Comment:
This should not be necessary with the recent addition of superjson. If you're finding it necessary with latest rolled in, let me know.

Reply:
yeah i think this stayed in as i was fixing merge conflicts. will remove